src/django_perf_rec/utils.py

```diff
@@ -91,7 +91,7 @@ def _get_details_from_pytest_request(frame: FrameType) -> TestDetails | None:
     return TestDetails(
         file_path=request.fspath.strpath,
         class_name=class_name,
-        test_name=request.function.__name__,
+        test_name=request.node.name,
```

Comment:
Compared to the solution I mentioned in #346, this works without depending on environment variables.

Reply:
Damn this code could be a lot smarter... the pytest plugin could save the `request` object so we wouldn't need stack inspection to retrieve it every time. A future cleanup, I guess.